docs/guides/security/monitoring/security-auditing-with-lynis/index.md

```diff
@@ -0,0 +1,161 @@
+---
+author:
+  name: Hackersploit
+description: 'Learn to perform a security audit on your system using Lynis.'
+og_description: 'Learn to perform a security audit on your system using Lynis.'
+keywords: ["linux log monitor", "linux system monitoring tools", "system monitor linux"]
+tags: ["monitoring","security"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-10-29
+modified: 2020-10-29
+modified_by:
+  name: Linode
+title: 'Automated Security Auditing with Lynis'
+h1_title: 'Security Auditing with Lynis'
+---
```

Comment:
You'll need to add a `slug` frontmatter here. It could probably  be "security-auditing-with-lynis"